tests/unit/TooltipTest.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```

Comment:
Should move this file to `tests/ui/TooltipTest.js` instead of `tests/unit/TooltipTest.js`